src/components/Users/UserSection.jsx

```diff
@@ -23,12 +29,26 @@ const UserSection = ({ loadingUsers, loadingActive }) => {
       {loadingUsers ? (
         <UsersList loadingActive={loadingActive} />
       ) : (
-        <section className="panel">
-          <strong>Users List</strong>
-          <br />
-          <br />
-          <div style={{ marginBottom: '20px', textAlign: 'center' }}>Loading users list...</div>
-        </section>
+        <Container>
+          <Box
+            sx={{
+              marginTop: 3,
+              minWidth: 120,
+              display: 'flex',
+              flexDirection: 'column'
+            }}
+          >
+            <Paper elevation={2} sx={{ display: 'inline-block', mx: '2px', padding: '20px' }}>
+              <Typography sx={{ marginBottom: '20px', textAlign: 'center' }}>
```

Comment:
The `<Typography>` component in Line 42 will trigger an error since there's a `<div>` inside it at Line 46. Could replace this with a `<Box>`.